styles/styles.scss

```diff
@@ -1,12 +1,119 @@
+$base: 8px;
```

Comment:
I think something got messed up here? breakpoints and colours have their own files now (see the import statements on line 92